src/measure_theory/bochner_integration.lean

```diff
@@ -37,11 +37,32 @@ Step 5: Define the Bochner integral on L1 functions by extending the integral on
 
 ## Main statements
 
-• Basic properties of the Bochner integral.
+• Basic properties of the Bochner integral on functions of type `α → β`, where `α` is a measure
+  space and `β` is a real normed space.
 
-• The Lebesgue dominated convergence theorem : `tendsto_integral_of_dominated_convergence`
+`integral_zero`                  : `∫ 0 = 0`
```

Comment:
Please turn this into a markdown list. Otherwise the rendered docs will look funny. So prepend `*`s
```
* `integral_zero`
* ...
```